src/OpenTelemetry/Internal/SdkHealthReporter.cs

```diff
@@ -0,0 +1,65 @@
+// <copyright file="SdkHealthReporter.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+#nullable enable
+
+using System.Diagnostics.Metrics;
+
+namespace OpenTelemetry.Internal
+{
+    internal sealed class SdkHealthReporter
+    {
+        private static readonly Dictionary<int, Func<Measurement<long>>> MeasurementDroppedCountCallbacks = new();
+
+        private static readonly Meter InternalMeter = new Meter("OpenTelemetry.Sdk");
+
+        private static readonly ObservableCounter<long> MeasurementDroppedCountObs = InternalMeter.CreateObservableCounter("otel.dotnet.sdk.measurement.dropped_count", GetMeasurementDroppedCounts);
```

Comment:
We should avoid any SDK specific features such as `Views`, `SetMaxMetrics`, `SetMaxMetricStreamPoints` to be applied to any of our internal metrics. This would have to be done in the `MeterProviderSdk`.